hubstorage/resourcetype.py

```diff
@@ -26,7 +28,9 @@ def _iter_lines(self, _path, **kwargs):
 
         r = self.client.request(**kwargs)
 
-        return r.iter_lines()
+        lines = r.iter_lines()
+        return (l.decode('utf8') if isinstance(l, six.binary_type)
+                else l for l in lines)
```

Comment:
it looks like this method returned generator that produced strings, and now it produces unicode. why is that?